integration_tests/e2e/vc2GoalsPage.cy.ts

```diff
@@ -14,7 +14,7 @@ context('Prisoners VC2 goals page', () => {
 
   it('should contain a banner about learning and work progress service replacing VC2', () => {
     // Given
-    cy.setupWorkAndSkillsPageStubs({ prisonerNumber: 'G6123VU', emptyStates: false })
+    cy.setupWorkAndSkillsPageStubs({ prisonerNumber: 'G6123VU' })
```

Comment:
I've set `emptyStates: false` as the default (further down), so no need for calls to this stub to pass it (unless a given test wants to change the behaviour to get an "empty state")